dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ClientCore.cs

```diff
@@ -504,18 +500,6 @@ internal static OpenAIClientOptions GetOpenAIClientOptions(HttpClient? httpClien
     /// <returns>Chat object</returns>
     internal static OpenAIChatHistory CreateNewChat(string? text = null, OpenAIPromptExecutionSettings? executionSettings = null)
     {
-        // If text is not provided, create an empty chat with the system prompt if provided
-        if (string.IsNullOrWhiteSpace(text))
-        {
-            return new OpenAIChatHistory(executionSettings?.ChatSystemPrompt);
-        }
-
-        // Try to parse the text as a chat history
-        if (XmlPromptParser.TryParse(text!, out var nodes) && ChatPromptParser.TryParse(nodes, out var chatHistory))
```

Comment:
@RogerBarreto with this deletion `XmlPromptParser` is not used anymore, when you do `kernel.InvokeAsync` or `function.InvokeAsync`. Which means that XML prompt syntax (`<message>`) does not work as well. This caused this issue: https://github.com/microsoft/semantic-kernel/issues/3960